elixir-format.el

```diff
@@ -125,54 +125,76 @@ Shamelessly stolen from go-mode (https://github.com/dominikh/go-mode.el)"
               (error "Invalid rcs patch or internal error in elixir-format--apply-rcs-patch"))))))))
   )
 
+(defun elixir-mix-path-set? ()
```

Comment:
Please rename this `elixir-format-mix-path-set-p`: the convention is to prefix with the namespace, and the `-p` means that this function is a predicate, that is, it returns a boolean value.

Reply:
done.